source/community/reactnative/package-lock.json

```diff
@@ -369,6 +369,189 @@
         "@babel/core": "^7.0.0-0"
       }
     },
+<<<<<<< HEAD
```

Comment:
package-lock file has merge conflict. Please resolve

Reply:
It happened by mistake. I resolved it using the current version. Now it's not appearing in the `files changed` section.